frontend/package.json

```diff
@@ -15,7 +15,7 @@
         "test-coverage": "vitest run --coverage"
     },
     "dependencies": {
-        "@bcgov-nr/nr-theme": "^1.7.1",
+        "@bcgov-nr/nr-theme": "^v1.8.0",
```

Comment:
this is not correct, we shouldn't include "v", just "1.8.0". might just be a typo

Reply:
already fixed